packaging/darwin/Distribution.in

```diff
@@ -39,14 +31,9 @@ function installCheck() {
     </script>
     <choices-outline>
         <line choice="crc"/>
-        <line choice="crc-tray"/>
```

Comment:
There might be slightly more changes needed in this area to hide the "customize install" button if it's still present. This was added in https://github.com/crc-org/crc/commit/ba0b84686c41d5a451025414f0dcc8f472ada047
This can be done in a follow-up as this should not cause any breakage if still present.